xenmgr/Vm/Actions.hs

```diff
@@ -479,11 +479,11 @@ getVhdReferences vhd = concat <$> (mapM (diskVhdReferences vhd) =<< getVms) wher
   diskVhdReferences vhd vm = zip (repeat vm) . filter (references vhd) . M.elems <$> getDisks vm where
     references vhd disk = diskPath disk == vhd
 
-startVm :: Uuid -> XM ()
-startVm uuid = do
+startVm :: Bool -> Uuid -> XM ()
+startVm is_reboot uuid = do
```

Comment:
What do you think about doing:
```
startVm uuid = do _startVM False uuid
restartVm uuid = do _startVm True uuid
_startVm is_reboot uuid = do...
```
That way the assorted startVm False and startVm True are replaced by startVm and restartVm and you don't have to know what the boolean argument means.

Reply:
Yea wrapping that startVm call is probably the way to go to make this more readable.  I like it.